lib/hanami/action/exposable/guard.rb

```diff
@@ -0,0 +1,99 @@
+module Hanami
+  module Action
+    module Exposable
+      # Guard for Exposures API.
+      # Prevents exposure of reserved words
+      #
+      # @since x.x.x
+      #
+      # @see Hanami::Action::Exposable::Guard::ClassMethods#expose
+      # @see Hanami::Action::Exposable::Guard::ClassMethods#reserved_word?
+      module Guard
+        # Override Ruby's hook for modules.
+        # It prepends a guard for the exposures logic
+        #
+        # @param base [Class] the target action
+        #
+        # @since x.x.x
+        # @api private
+        #
+        # @see http://www.ruby-doc.org/core-2.1.2/Module.html#method-i-included
+        def self.included(base)
+          class << base
+            prepend ClassMethods
+          end
+        end
+
+        # Exposure of reserved words
+        #
+        # @since x.x.x
+        class IllegalExposeError < ::StandardError
+        end
+
+        # Exposures API Guard class methods
+        #
+        # @since x.x.x
+        # @api private
+        module ClassMethods
+          # Prevents exposure if names contain a reserved word.
+          #
+          # @param names [Array<Symbol>] the name(s) of the attribute(s) to be
+          #   exposed
+          #
+          # @return [void]
+          #
+          # @since x.x.x
+          def expose(*names)
+            detect_reserved_words!(names)
+
+            super
+          end
+
+          private
+
+          # Raises error if given names  contain a reserved word.
+          #
+          # @param names [Array<Symbol>] a list of names to be checked.
+          #
+          # @return [void]
+          #
+          # @raise [IllegalExposeError] if names contain one or more of reserved
+          #   words
+          #
+          # @since x.x.x
+          # @api private
+          def detect_reserved_words!(names)
+            names.each do |name|
+              if reserved_word?(name)
+                error_text = "#{name} is a reserved word. It cannot be exposured"
+                raise IllegalExposeError, error_text
```

Comment:
We don't need to define `error_text` here. You can break it across two lines after the comma to keep it under 80 characters
```
  raise IllegalExposeError, 
    "..."
```